routes/socket/game/start-game.js

```diff
@@ -525,6 +525,24 @@ const beginGame = game => {
 		},
 		process.env.NODE_ENV === 'development' ? 100 : experiencedMode ? 5400 : 9000
 	);
+
+	for (let affectedPlayerNumber = 0; affectedPlayerNumber < game.publicPlayersState.length; affectedPlayerNumber ++) {
+		const affectedSocketId = Object.keys(io.sockets.sockets).find(
+			socketId =>
+				io.sockets.sockets[socketId].handshake.session.passport &&
+				io.sockets.sockets[socketId].handshake.session.passport.user === game.publicPlayersState[affectedPlayerNumber].userName
+		);
+		player.pingTime = Date.now();

```

Comment:
```suggestion

```

This is unnecessary as its an automatic ping and we don't need to set a ping timer for any player (also this will error, since player is undefined)